schema-next.yaml

```diff
@@ -2,6 +2,12 @@ file_format: 1.1.0
 schema_url: https://opentelemetry.io/schemas/next
 versions:
   next:
+    all:
```

Comment:
should here be also noted changes to the metrics?

Reply:
Initially it was applied only to metrics but I made it to apply to all as per https://github.com/open-telemetry/semantic-conventions/pull/1632#discussion_r1867013714